specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-03-01-preview/WorkspaceManagerAssignments.json

```diff
@@ -0,0 +1,734 @@
+{
+  "swagger": "2.0",
```

Comment:
>swagger [](http://example.com/codeflow?start=3&length=7)

please address all the errors listed in the automated checks 


[https://github.com/Azure/azure-rest-api-specs/pull/21273/checks?check_run_id=11314998198](https://github.com/Azure/azure-rest-api-specs/pull/21273/checks?check_run_id=11314998198)